src/domain/audit/repositories/class.audit.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace leantime\domain\repositories {
+
+    use leantime\core;
+    use pdo;
+
+    class audit
+    {
+
+        public function __construct()
+        {
+
+            $this->db = core\db::getInstance();
+        }
+
+        public function storeEvent($action = "ping", $values = "", $entity = "" , $entityId = 0, $userId = 0, $projectId = 0, $thedate = '')
+        {
+           
+            if ( $thedate == '') 
+            {
+                $thedate2 = date('Y-m-d H:i:s');
```

Comment:
no need for the else, if you just set $thedate instead of changing it to $thedate2.

` 
  if ($thedate == '')
  {
      $thedate = date('Y-m-d H:i:s');
  }
`

Reply:
This is an old coding habit catched with languages for which function params are readonly.
Which is not the case with PHP though I prefer to avoid changing the values from function parameters : it's more clear in my head this way.